### PR TITLE
0077 JavaScript Code Samples 2

### PR DESCRIPTION
**Pull request by pdp2121**
Submitting a bounty to add more code samples and fix depreciated ones to capture more common tasks in JavaScript.

### CODE REVIEW

File: open_bounties/0077 JavaScript Code Samples 2.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# JavaScript Code Samples 2
+
+## Overview
+
+Currently, our JavaScript documentations have [code snippets](https://github.com/XRPLF/xrpl.js/tree/main/packages/xrpl/snippets/src) and [code samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples) for some common usecases. However, there needs to be more examples to capture common tasks that users would want the library to do, and fixes to outdated or misconfigured snippets. Developers would benefit from additional relevant, high quality, well-documented examples in JavaScript.
+
+## Details
+
+To add missing code samples, follow the instructions at the bottom of the page, under **Contribute Code Samples**.
```
Comment (jonathanlei) [inline open_bounties/0077 JavaScript Code Samples 2.md:17]: I thought this is to add to [code snippets](https://github.com/XRPLF/xrpl.js/tree/main/packages/xrpl/snippets/src)?

**Comment by wojake** [comment]
I suppose milestone 3 should be around $2500 since it's a bigger job as compared to the other milestones, I will not pursue it tho :)

File: open_bounties/0077 JavaScript Code Samples 2.md (hunk 1 above)
Comment (pdp2121) [reply open_bounties/0077 JavaScript Code Samples 2.md:17]: Some of the changes should be in xrpl-dev-portal (like changing the depreciated codes), so my initial thought would be to have all of them in xrpl-dev-portal first then move them around later if needed. We can also add a column (Sample/Snippet) to point each of them to a location we want, but the difference between snippets in xrpl-js and code sample in xrpl.org is kinda slim so I'm not sure. @mDuo13 what do you think?

**Comment by pdp2121** [comment]
> I suppose milestone 3 should be around $2500 since it's a bigger job as compared to the other milestones, I will not pursue it tho :) . Same goes to milestone 5 @ ~$2000 as well.

Thanks @wojake! I will need budget confirmation first then will adjust the award prize. But yea those 2 should take more effort, even though we have Python version as guideline.

**Review by jonathanlei — APPROVED** [review]

**Review by ckniffen — APPROVED** [review]

File: open_bounties/0077 JavaScript Code Samples 2.md (hunk 1 above)
Comment (mDuo13) [reply open_bounties/0077 JavaScript Code Samples 2.md:19]: Makes sense to me. Loosely, I think I originally pictured the dev portal list of code samples as being bigger / more complete snippets or examples of how to use the ledger in general, while the in-repo snippets were smaller bits or more specific to the library. In practice, I'm now of the opinion that that distinction is too fuzzy to be useful, and it's fine to put all of them in the dev portal.

**Review by intelliot — APPROVED** [review]

**Pull request merged**